storage/s3.go

```diff
@@ -86,7 +89,7 @@ func (s *S3) PutFile(content io.ReadSeeker, filepath string, contentType string)
 }
 
 func (s *S3) GetFile(filepath string, headers http.Header) (*controller.File, *controller.APIError) {
-	object, err := s.session.GetObject(
+	object, err := s.client.GetObject(s.ctx,
```

Comment:
context should be the first argument of the method instead so it can be propagated from the request. Same for other methods.

Reply:
agree, added